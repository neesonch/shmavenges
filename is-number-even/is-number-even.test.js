const isNumberEven = require('./is-number-even.js');

test('Even integer', () => {
  expect(isNumberEven(22)).toMatchSnapshot();
});
