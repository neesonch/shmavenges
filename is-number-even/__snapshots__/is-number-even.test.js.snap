// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Even integer 1`] = `true`;
